src/Avatar/Avatar.js

```diff
@@ -60,13 +88,19 @@ Avatar.propTypes = {
   className: PropTypes.string,
   /** Applied as data-hook HTML attribute that can be used to create an Avatar driver in testing */
   dataHook: PropTypes.string,
+  /** Avatar presence. Options like 'online' mean that the conatct is online */
+  presence: PropTypes.oneOf(['online', 'offline', 'busy']),
+  /** A node to be rendered as Indication.*/
+  indication: PropTypes.node,
+  /** function which triggered on indication click.*/
```

Comment:
Worth mentioning the following, IMO:
```suggestion
  /** function which triggered on indication click. Notice it will be clickable just in case `indication` is provided */
```